### ISSUE TITLE
scanner problems

### ISSUE THREAD

**Issue by allinduetime** [EMCJava/WuWaOpt#12]
accidentally pressed enter
anyways the scanner doesnt scan properly and seems to mess up most commonly when scrolling, due to the mouse movements being jerky.

**Comment by EMCJava** [comment]
what scan delay did you use, have you tried putting in larger number? (around 300)

**Comment by allinduetime** [comment]
Went all the way to 500, still had the same effect
________________________________

**Comment by EMCJava** [comment]
Hmm, have you try turning off Nvidia's filter, HDR, any software that can alter the color of the game?

**Comment by allinduetime** [comment]
I don't think that would affect the inconsistency of the mouse movements, nor how far it scrolls.
________________________________

**Comment by allinduetime** [comment]
But also I don't have any of those on, as far as I'm aware.
________________________________

**Comment by EMCJava** [comment]
I see, is it scrolling too much, too little, or just by random?

**Comment by allinduetime** [comment]
Scrolling too little, and the mouse movements keep shifting way too high
________________________________

**Comment by EMCJava** [comment]
Could you try replacing WuWaScan.exe from version 1.4.0 with this? I have reduced the mouse scrolling and movement speed to see if this helps
[WuWaScan.zip](https://github.com/user-attachments/files/18281109/WuWaScan.zip)

**Comment by allinduetime** [comment]
Sorry about the late response I didn't see this email til now, I'll try this in a bit
________________________________

**Comment by EMCJava** [comment]
It's okay, but the 2.0 update changed some of the UI, the tweaked scanner won't work properly, you can still test the scanner to see if it's scrolling correctly.

**Comment by allinduetime** [comment]
this one weirdly enough doesn't even load past the beginning chunk

**Comment by allinduetime** [comment]
https://github.com/user-attachments/assets/4293a664-2fcc-48d3-8ca0-80576e245e38
Ignore the sound but example issue from the newest update

**Comment by EMCJava** [comment]
The game didn't seem to register all scrolling inputs, may I ask what is the maximum framerate you put in the settings?

**Comment by allinduetime** [comment]
I'll have to check but I suspect it's set to 30fps, I can try to make sure it's on 60 tonight if I'm right on that. This was on 500 ms again btw
________________________________

**Comment by allinduetime** [comment]
running it on 60 did fix the issue!

**Comment by allinduetime** [comment]
i mean the movements are still inaccurate but itdoesnt miss now

**Comment by EMCJava** [comment]
I'm going to close this for now because implementing “interactive scrolling” will take too much time. As for the inaccurate movement, it's intentional, and if it gets the job done, I mean...
